library/core/src/num/f32.rs

```diff
@@ -443,6 +443,15 @@ impl f32 {
     #[stable(feature = "assoc_int_consts", since = "1.43.0")]
     pub const MAX: f32 = 3.40282347e+38_f32;
 
+    
+    /// Largest negative normal `f32` value.
+    ///
+    /// Equal to &minus;2<sup>[`MIN_EXP`]&nbsp;&minus;&nbsp;1</sup>.
+    ///
+    /// [`MIN_EXP`]: f32::MIN_EXP
+    #[stable(feature = "assoc_int_consts", since = "1.43.0")]
+    pub const MAX_NEGATIVE: f32 = -1.17549435e-38_f32;
```

Comment:
How could this constant be stable since 1.43 when you are adding it now? :)

New API needs an `unstable` gate first, and is moved to stable later.